boards/posix/native_posix/doc/index.rst

```diff
@@ -626,3 +629,38 @@ development by integrating more seamlessly with the host operating system:
   A backend/"bottom" for Zephyr's CTF tracing subsystem which writes the tracing
   data to a file in the host filesystem.
   More information can be found in :ref:`Common Tracing Format <ctf>`
+
+Host based flash access
+***********************
+
+If a flash device is present, the file system partitions on the flash
+device can be exposed through the host file system by enabling
+:option:`CONFIG_FUSE_FS_ACCESS`. This option enables a FUSE
+(File system in User space) layer that maps the Zephyr file system calls to
+the required UNIX file system calls, and provides access to the flash file
+system partitions with normal operating system commands such as ``cd``,
+ ``ls`` and ``mkdir``.
```

Comment:
``doc/_build/rst/doc/boards/posix/native_posix/doc/index.rst:642: WARNING: Unexpected indentation.``

Reply:
@vanwinkeljan This is how it looks: https://builds.zephyrproject.org/zephyrproject-rtos/zephyr/16467/docs/boards/posix/native_posix/doc/index.html#host-based-flash-access

Reply:
P.S.  using ``make htmldocs-fast`` avoids creating all the options documentation details and runs in a quarter of the time as the full doc build.

Reply:
@dbkinder is this a new make target, as I get `No rule to make target`?